python/cuml/common/base.pyx

```diff
@@ -152,3 +207,31 @@ class Base:
     def __setstate__(self, state):
         self.__dict__.update(state)
         self.handle = cuml.common.handle.Handle()
+
+    def __getattr__(self, attr):
+        real_name = '_' + attr
```

Comment:
do we only apply magic to fields ending in `_` as well?

Reply:
no, it only happens for fields that start with `_` and also are cuml arrays, in case we want to have attributes that don't end in `_` for whatever reason (not sure if every array attribute in the scikit API does end with an underscore)